pages/common/sublist3r.md

```diff
@@ -0,0 +1,20 @@
+# sublist3r
+
+> Fast subdomains enumeration tool for penetration testers.
+> More information: <https://github.com/aboul3la/Sublist3r>.
+
+- Find subdomains for a domain:
+
+`sublist3r --domain {{domain_name}}`
+
+- Find subdomains for a domain, also use brute force:
+
+`sublist3r --domain {{domain_name}} --bruteforce`
+
+- Save found results to a text file:
+
+`sublist3r --domain {{domain_name}} --output {{output_file}}`
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
`sublist3r --domain {{domain_name}} --output {{path/to/output_file}}`
```